grr/server/grr_response_server/databases/spanner.py

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
```suggestion
#!/usr/bin/env python
"""Spanner implementation of the GRR relational database abstraction.

See grr/server/db.py for interface.
"""
```
Nit: https://google.github.io/styleguide/pyguide.html#382-modules